src/components/Class/ClassList/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from 'react-redux';
+import moment from 'moment';
+import { actions as classActions } from '../redux/action'
+import Loader from '../../common/Loader'
+import "./styles.scss";
+function ClassList() {
```

Comment:
Could we put 2 blank lines before the component starts?

Reply:
done